apps/meteor/client/views/omnichannel/managers/ManagersTable.tsx

```diff
@@ -88,7 +100,7 @@ const ManagersTable = () => {
 			)}
 			{isSuccess && data.users.length > 0 && (
 				<>
-					<GenericTable>
+					<GenericTable aria-busy={text !== debouncedText} aria-live='assertive'>
```

Comment:
why `aria-live`?

Reply:
I usually use `aria-live` to indicate that this area is `live` and prone to changes, shouldn't we be using it like that